crates/sui-indexer/src/lib.rs

```diff
@@ -320,7 +320,7 @@ struct PgConectionPoolConfig {
 impl PgConectionPoolConfig {
     const DEFAULT_POOL_SIZE: u32 = 100;
     const DEFAULT_CONNECTION_TIMEOUT: Duration = Duration::from_secs(30);
-    const DEFAULT_STATEMENT_TIMEOUT: Duration = Duration::from_secs(5);
+    const DEFAULT_STATEMENT_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Can we change this such that only the writer has a higher statement timeout?

Reply:
let's make this an env-var that we only set in writer instances